numpy/lib/function_base.py

```diff
@@ -4289,14 +4316,12 @@ def _percentile(a, q, axis=None, out=None,
     # avoid expensive reductions, relevant for arrays with < O(1000) elements
     if q.size < 10:
         for i in range(q.size):
-            if q[i] < 0. or q[i] > 100.:
+            if q[i] < 0. or q[i] > 1.:
                 raise ValueError("Percentiles must be in the range [0,100]")
```

Comment:
Error message is shared between percentile and quantile. Should it mention both, or should we somehow taylor it?